_posts/homepage/2018-04-12-Sprints.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+image: stickers.jpg
+title: Sprints
+description: PyData Ams
+category: [homepage]
+---
+
+For the first time in its' history, PyData Amsterdam will host Open Source Contribution Sprints. These will take place
+on Monday the 15th.
+
+Our sprints are a day of intensive learning and development on an open source project of your choice.
+It’s a time to come together, to share what you’ve learned during the conference with the people you met, 
```

Comment:
'your choice' -> based on the options we provide, right?

Reply:
yes, but I couldn't think of a nice way of writing that down. At some point we should add a more detailed page for the sprints that includes the venue and projects

Reply:
I think what this page is missing is something along the lines of 'collaborating' or 'contributing'. It's focussed on learning, while I think we'd want to emphasise it's also open to people who are just eager to contribute to open source together. 

Inspiration: https://de.pycon.org/sprints/

Reply:
The thing I dislike about the pycon wording is that it can make people feel that the sprints are not for them, because it focuses on maintainers. I do agree that currently it's too focused on learning. I made some changes to reflect that

Reply:
If we move lines 16-17 up to the first paragraph, so it leads with us bringing together eager developers/maintainers of [...] libraries [...] and we reword as follows, I guess the intention becomes clearer -- i.e. let's bring in people and work (primarily) on the projects they maintain.

> Our sprints are a day of intensive development on open source projects

Reply:
👍 done